docs/dev/rfc/odfe-cli-rest-api-as-commands.md

```diff
@@ -0,0 +1,302 @@
+
+## Available commands
+
+1. [get command](./odfe-cli-rest-api-as-commands.md#1-get-command)
+2. [post command](./odfe-cli-rest-api-as-commands.md#2-post-command)
+3. [put command](./odfe-cli-rest-api-as-commands.md#3-put-command)
+4. [delete command](./odfe-cli-rest-api-as-commands.md#4-delete-command)
+
+## Common parameters
+
+The following options can be applied to all of the odfe-cli commands. These parameters are supported by Elasticsearch as mentioned [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html).
+In the future we will support more parameters or include multiple values that can be added as flag to any commands.
+
+
+
+|Parameter name|Short hand	|Default|Usage|
+|---------------|-----------|-------|-------|
+|--pretty	    |-b	        | false	|if true, the response returned will be pretty formatted	|
+|--output-format |-o	|json	|other value is yaml. The result to be returned in the (if supported) more readable yaml format.	|
+|--human	| N/A	| false	|the output statistics are returned in a human friendly format. (e.g. `"exists_time": "1h"` or `"size": "1kb"`) instead of (e.g. `"exists_time_in_millis": 3600000` or `"size_in_bytes": 1024`). 	|
+|--filter-path	|-f	| N/A	|filter the response fields returned by cluster. It accepts a comma separated list of filters expressed with the dot notation.  eg: `--filter-path=“took,hits.hits._id,hits.hits._score”`	|
+
+
+
+## 1. get command
+
+### Description
+
+Use GET API to execute requests against Elasticsearch cluster. This command enables you to run any GET based REST API commands across all 
+categories.
+
+### Synopsis
+
+```
+odfe-cli curl get options [common parameters]
+```
+
+### Options
+
+|Name|Required|Short hand|Usage|Separator for multiple values|
+|---| :---: |:---:|---	|:---:|
+| --path	|Y	|-P	|url-path	|N/A	|
+| --query-params|N	|-q	|url query parameters	|&	|
+| --headers	 |N	|-H	|pass additional information with request. It consists of case-insensitive name followed by a colon (`:`), then by its value.	|semi colon ( ; )	|
+|--data	|N	|-d	|Sends the specified data in the command to Elasticsearch". If value starts with the letter @, the rest should be a file name to read the data from.	|N/A	|
+|--data-binary	|N	|-b	|This is similar to "--data", except that newlines and carriage returns are preserved and conversions are never done. This is required if input file is compressed.	|N/A	|
+|--help	|N	|-h	|Help for get command	|N/A	|
+
+### Common Parameters
+
+see [here](./odfe-cli-rest-api-as-commands.md#common-parameters)
+
+
+**Note: users can escape separator like ‘\\;’ to use as part of value.**
+
+### Example:
+
+1. To get document count for an index
+
+```
+odfe-cli curl get --path "_cat/count/my-index-01" \
+                 --query-params "v=true" \
+                 --pretty
+```
+
+Response
+
+```
+epoch         timestamp        count
+1612311592    12:24:24          100
+```
+2.  To  returns the health status of a cluster.
```

Comment:
return not returns. Remove period at end of line.

Reply:
Ack

Reply:
Ack